lms/static/scripts/frontend_apps/components/Button.js

```diff
@@ -3,10 +3,11 @@ import { createElement } from 'preact';
 
 /**
  * @typedef ButtonProps
+ * @prop {Object} [buttonRef]
  * @prop {string} [className]
  * @prop {boolean} [disabled]
  * @prop {string} label
- * @prop {() => any} onClick
+ * @prop {(e: Event) => any} onClick
```

Comment:
What was the `onClick` change required for here?

Reply:
I think it's not needed. Going to remove it.